Contributing.md

```diff
@@ -13,6 +17,25 @@ detect the projects and import it properly.
 3. If you discover an error on `pom.xml` after import about Tycho, you can use Quick Fix
 (Ctrl+1) to install the Tycho Maven integration.
 
+## Local debug flow with an editor
+
+Eclipse desktop allows to debug Language Server being written in the same workspace. To do that:
```

Comment:
Eclipse desktop allows developers to debug Language Server [code/projects?] being written in the same workspace.

Reply:
https://github.com/camel-tooling/camel-language-server/pull/468